pip/requirements.txt

```diff
@@ -251,6 +253,7 @@ python-cjson==1.2.2;python_version<'3.0'
 python-daemon==2.2.4
 python-dateutil==2.8.1
 python-ldap==3.3.1
+python-rapidjson==1.0;python_version>'3.0'
```

Comment:
Does this need to be there to build? The rapidjson headers should have been shipped with the correctionlib sdist, and we don't use rapidjson on the python side at all.